translations/Marathi/special-terms.md

```diff
@@ -0,0 +1,42 @@
+﻿Terms
+
+Message : संदेश , Most appropriate synonym for general usage in zulip, <username> ला एक संदेश पाठवा
+Private Message (PM) : खाजगी संदेश , Best translation of private to imply it's meaning , <username> ला खासगी एक संदेश पाठवा
+Group PM : ग्रुप खाजगी संदेश  , Transliteration used for group , <group name> मध्ये एक ग्रुप खाजगी संदेश पाठवा
+Realm : संघटन , used translation of organization for same reason as of chinese translation , एक संघटन बनवा
+Stream : स्ट्रीम  , used transliteration for better understanding , स्ट्रीममध्ये प्रवेश करा
+Topic : विषय , best word to signify the usage in zulip , नवीन विषय सुरू करा
+Private/Invite-Only Stream : खाजगी / केवळ-आमंत्रित स्ट्रीम , used translation of "private/invited-only" for better understanding , आपले खाजगी / केवळ-आमंत्रित स्ट्रीम
+Public Stream : सार्वजनिक स्ट्रीम  , best translation of public for usage in zulip , आपले सार्वजनिक स्ट्रीम 
+Bot : बॉट , used transliteration , स्वागत बॉट
+Integration : अनुप्रयोग एकीकरण , used translation of "application integration" , फेसबुक सह झुलीप अनुप्रयोग एकात्मता
+Notification : सूचना , translates to information but is understandable by user , सूचना सक्रिय करा
+Alert Word : सतर्क शब्द , only translation which is understandable by average user , आपले सतर्क शब्द
+View : पहा , used translation of see for implication that it is verb , संदेश पहा
+Home : मुख्यपृष्ठ , used translation of homepage instead for better understanding , मुख्यपृष्ठावर जा
```

Comment:
It should be 'मुखपृष्ठ', not 'मुख्यपृष्ठ'

Reply:
मुखपृष्ठ is 'front page' in hindi and front page would generally be used in terms of books, magazines,etc